src/qt/bitcoin.cpp

```diff
@@ -758,8 +758,8 @@ int GuiMain(int argc, char* argv[])
     }
     // Validate/set custom css directory
     if (gArgs.IsArgSet("-custom-css-dir")) {
-        fs::path customDir = fs::path(gArgs.GetArg("-custom-css-dir", ""));
-        QString strCustomDir = QString::fromStdString(customDir.string());
+        fs::path customDir = fs::PathFromString(gArgs.GetArg("-custom-css-dir", ""));
+        QString strCustomDir = QString::fromStdString(fs::PathToString(customDir));
```

Comment:
Should be `PathToQString`` used instead of `QString::fromStdString(fs::PathToString))` ?

Reply:
Resolved in latest push!